### PR TITLE
Track Scan request batch submit event via app insights

### PR DESCRIPTION
**Pull request by haonliu**
#### Description of changes

image

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/logger/src/logger-event-measurements.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    maxParallelTasksPossible: number;
+}
+
+export interface ScanTaskStartedMeasurements extends BaseTelemetryMeasurements {
+    scanWaitTime: number;
+    scanTaskStartTime: number;
+}
+
+export interface ScanTaskCompletedMeasurements extends BaseTelemetryMeasurements {
+    scanCompleteTime: number;
```
Comment (lamaks) [inline packages/logger/src/logger-event-measurements.ts]: scanCompletedTime
scanExecutionTime
scanWallClockTIme

File: packages/logger/src/logger-event-measurements.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export interface BatchScanRequestMeasurements extends BaseTelemetryMeasurements {
+    totalScanRequests: number;
+    acceptedScanRequests: number;
+    rejectedScanRequests: number;
+}
+
+export interface BatchPoolMeasurements extends BaseTelemetryMeasurements {
+    runningTasks: number;
+    samplingIntervalInSeconds: number;
+    maxParallelTasksPossible: number;
```
Comment (lamaks) [inline packages/logger/src/logger-event-measurements.ts]: this is max actual running task, right?
should be just maxParallelTasks?

File: packages/logger/src/logger-event-measurements.ts (hunk 1 above)
Comment (haonliu) [reply packages/logger/src/logger-event-measurements.ts]: fixed

File: packages/logger/src/logger-event-measurements.ts (hunk 2 above)
Comment (haonliu) [reply packages/logger/src/logger-event-measurements.ts]: fixed

**Pull request merged**